src/compiler/compile/nodes/Element.ts

```diff
@@ -603,6 +612,21 @@ export default class Element extends Node {
 							}
 						}
 
+						// interactive-supports-focus
+						if (
+							!has_disabled_attribute(attribute_map) &&
+							!is_hidden_from_screen_reader(this.name, attribute_map) &&
+							!is_presentation_role(current_role) &&
+							is_interactive_roles(current_role) &&
+							is_static_element(this.name, attribute_map) &&
```

Comment:
[memo]

This element doesn't throw the warning because the `<span>` is not static element.

```svelte
<script>
  const role = "button";
</script>

<span on:click={() => {}} {role}>Submit</span>
```